src/feedback/StarRating.tsx

```diff
@@ -1,97 +1,180 @@
 import { Fragment, FunctionComponent, h } from "preact";
+import { useRef } from "preact/hooks";
 
 import { Dissatisfied } from "./icons/Dissatisfied";
 import { Neutral } from "./icons/Neutral";
 import { Satisfied } from "./icons/Satisfied";
 import { VeryDissatisfied } from "./icons/VeryDissatisfied";
 import { VerySatisfied } from "./icons/VerySatisfied";
+import { arrow, offset, useFloating } from "./packages/floating-ui-preact-dom";
 import { FeedbackTranslations } from "./types";
 
 const scores = [
   {
-    color: "var(--bucket-feedback-dialog-rating-1-color, #dd6b20)",
-    bg: "var(--bucket-feedback-dialog-rating-1-background-color, #fbd38d)",
-    icon: <VeryDissatisfied />,
-    getLabel: (t: FeedbackTranslations) => t.ScoreVeryDissatisfiedLabel,
+    color: "var(--bucket-feedback-dialog-rating-5-color, #38a169)",
+    bg: "var(--bucket-feedback-dialog-rating-5-background-color, #9ae6b4)",
+    icon: <VerySatisfied />,
+    getLabel: (t: FeedbackTranslations) => t.ScoreVerySatisfiedLabel,
+    value: 5,
   },
   {
-    color: "var(--bucket-feedback-dialog-rating-2-color, #ed8936)",
-    bg: "var(--bucket-feedback-dialog-rating-2-background-color, #feebc8)",
-    icon: <Dissatisfied />,
-    getLabel: (t: FeedbackTranslations) => t.ScoreDissatisfiedLabel,
+    color: "var(--bucket-feedback-dialog-rating-4-color, #48bb78)",
+    bg: "var(--bucket-feedback-dialog-rating-4-background-color, #c6f6d5)",
+    icon: <Satisfied />,
+    getLabel: (t: FeedbackTranslations) => t.ScoreSatisfiedLabel,
+    value: 4,
   },
   {
     color: "var(--bucket-feedback-dialog-rating-3-color, #787c91)",
     bg: "var(--bucket-feedback-dialog-rating-3-background-color, #e9e9ed)",
     icon: <Neutral />,
     getLabel: (t: FeedbackTranslations) => t.ScoreNeutralLabel,
+    value: 3,
   },
   {
-    color: "var(--bucket-feedback-dialog-rating-4-color, #48bb78)",
-    bg: "var(--bucket-feedback-dialog-rating-4-background-color, #c6f6d5)",
-    icon: <Satisfied />,
-    getLabel: (t: FeedbackTranslations) => t.ScoreSatisfiedLabel,
+    color: "var(--bucket-feedback-dialog-rating-2-color, #ed8936)",
+    bg: "var(--bucket-feedback-dialog-rating-2-background-color, #feebc8)",
+    icon: <Dissatisfied />,
+    getLabel: (t: FeedbackTranslations) => t.ScoreDissatisfiedLabel,
+    value: 2,
   },
   {
-    color: "var(--bucket-feedback-dialog-rating-5-color, #38a169)",
-    bg: "var(--bucket-feedback-dialog-rating-5-background-color, #9ae6b4)",
-    icon: <VerySatisfied />,
-    getLabel: (t: FeedbackTranslations) => t.ScoreVerySatisfiedLabel,
+    color: "var(--bucket-feedback-dialog-rating-1-color, #dd6b20)",
+    bg: "var(--bucket-feedback-dialog-rating-1-background-color, #fbd38d)",
+    icon: <VeryDissatisfied />,
+    getLabel: (t: FeedbackTranslations) => t.ScoreVeryDissatisfiedLabel,
+    value: 1,
   },
-];
+] as const;
```

Comment:
https://linear.app/bucket/issue/BUC-1336/investigate-slim-version-of-modal-which-asks-for-score-then-comment#comment-addd4395

Reply:
Changed 👍